README.rst

```diff
@@ -1,130 +1,154 @@
-``pip3 install coala``. If you need more information about the installation and
-dependencies, take a look at our `installation documentation
-<http://coala.rtfd.org/en/latest/Users/Install.html>`__.
+To get the list of arguments and their meaning, run:
 
-The latest code from master is automatically deployed to PyPI as a
-development version. Get it with ``pip3 install coala --pre``.
+.. code-block:: bash
 
-|PyPI|
+    $ coala --help
 
-Authors
--------
+To run coala over your project, create a ``.coafile``, the project-specific
+configuration file that will store all your settings for coala. A sample
+``.coafile`` will look something like this:
 
-*coala* is maintained by a growing community. Please take a look at the
-meta information in `setup.py <setup.py>`__ for current maintainers.
+.. code-block:: bash
 
-Project Status
---------------
+    [Default]
+    files = src/**/*.py
+    bears = SpaceConsistencyBear
+    use_spaces = True
 
-|Linux Build Status| |Windows Build status|
+Store this file in the project's root directory and run coala:
 
-|Scrutinizer Code Quality| |codecov.io|
+.. code-block:: bash
 
-|Documentation Status| |Gitmate|
+    $ coala
 
-Newcomers Guide and Getting Involved
-------------------------------------
+Please read our
+`coafile specification <http://coala.readthedocs.io/en/latest/Users/coafile.html>`__
+to learn more.
 
-If you are new and would like to contribute, read our `Getting Involved Site
-<http://coala.readthedocs.org/en/latest/Getting_Involved/README.html>`__!
+If you want to quickly want to test some settings without a ``.coafile``, you
+can run:
 
-We appreciate any help! Feel free to message us on
-`gitter <https://gitter.im/coala-analyzer/coala>`__. If you have any
-questions we're happy to help you!
+.. code-block:: bash
 
+    $ coala --files=setup.py --bears=SpaceConsistencyBear -S use_spaces=True
 
-License
--------
+-----
 
-|AGPL|
+=======
+Support
+=======
 
-This code falls under the GNU Affero General Public License as published
-by the Free Software Foundation, either version 3 of the License, or (at
-your option) any later version.
+Feel free to contact us at our `Gitter page <https://gitter.im/coala-analyzer/coala>`__, we'd be happy to help!
 
-Please note that some files or content may be copied from other places.
-Most of them are GPL compatible. There is a small portion of code in the
-tests that falls under the Creative Commons license, see
-https://creativecommons.org/licenses/by-sa/3.0/deed.de for more
-information.
+|gitter|
+
+-----
+
+================
+Getting Involved
+================
+
+If you would like to contribute, please take a look at our
+`Getting Involved <http://coala.readthedocs.io/en/latest/Getting_Involved/README.html>`__
+page or ask us at our gitter channel. We also have a
+`newcomer guide <http://coala.readthedocs.io/en/latest/Getting_Involved/Newcomers.html>`__
+to help you get started in fixing an issue. We appreciate any help!
```

Comment:
should be more accent on the achieving of the newcomer guide, as it's helping users fix an issue themselves :D

so something like: ``We also have a newcomer guide, which helps you get started by fixing an issue yourself! We appreciate any help!``